recipe/meta.yaml

```
@@ -152,6 +153,91 @@ outputs:
       license: MIT
       license_file: LICENSE.TXT
 
+  - name: libgdal-arrow-parquet
+    script: build_arrow_parquet.sh  # [unix]
+    script: build_arrow_parquet.bat  # [win]
+    build:
+      # currently, openssl 3 only works for arrow>=10; this may change if certain changes are
+      # backported; tracked in e.g. https://github.com/conda-forge/arrow-cpp-feedstock/pull/874
+      skip: true  # [(openssl == "3")  and (arrow_cpp != "10.0.1")]
```

I think we could skip openssl 3 for now...

```suggestion
      # skip openssl 3 pending https://github.com/conda-forge/arrow-cpp-feedstock/issues/918;
      # alternatively only dependon arrow > 10 (conda seems to get confused with a combined skip)
      skip: true  # [openssl == "3"]
```